### ISSUE TITLE
Can I get more detailed file information from the Plugin hooks 

### ISSUE THREAD

**Issue by nanwy** [TypeStrong/fork-ts-checker-webpack-plugin#711]
```js
class MyWebpackPlugin {
  apply(compiler) {
    const hooks = ForkTsCheckerWebpackPlugin.getCompilerHooks(compiler);

    // log some message on waiting
    hooks.waiting.tap('MyPlugin', () => {
      console.log('waiting for issues');
    });
    // don't show warnings
    hooks.issues.tap('MyPlugin', (issues) =>
      issues.filter((issue) => issue.severity === 'error')
    );
  }
}
```

Currently available from issues:

1. origin
2.code
3.severity
4.message
5.file
6.location

Can I get more information about this error file like mtime, atime

**Comment by piotr-oles** [comment]
I don't think I will support this case. You can always use `fs.stat()` method to get mtime and atime, right?
